DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: PKPDsim
 Type: Package
 Title: Tools for Performing Pharmacokinetic-Pharmacodynamic Simulations
-Version: 1.2.1
+Version: 1.2.9999
```

Comment:
a little bit of a jump?

Reply:
whoops, didn't mean to commit that, I just changed it so that when I installed locally I could make sure I was using the correct version. will revert.